scripts/populate_placeholders.py

```diff
@@ -35,6 +35,7 @@ def main():
         "[[HYPERLINK_A028]]": "[A028 - Record triage outcome](#api-Default-a028-record-triage-outcome)",
         "[[HYPERLINK_A029]]": "[A029 - Available actions for user list](#api-Default-a029-available-actions-for-user-list)",
         "[[HYPERLINK_A030]]": "[A030 - Retrieve e-RS business functions](#api-Default-a030-retrieve-business-functions)",
+        "[[HYPERLINK_A038]]": "[A038 - Retrieve Appointment](#api-Default-a038-retrieve-appointment)",
```

Comment:
Hmm, I remember you asked a question around referencing this endpoint as currently we've not yet documented the retrieve appointment endpoint. Is this currently being looked at? It feels like documenting this endpoint should be a pre-requisite to this work going live really given it's being referenced.

Reply:
stuart is aware of this situation